test/witan/models/dem/ccm/core/projection_loop_test.clj

```diff
@@ -97,3 +101,13 @@
       (is (= (+ (get-popn (:latest-yr-popn popn-wo-deaths) :popn 2015 0 "F")
                 (get-popn (:net-migration popn-wo-deaths) :net-mig 0 "F"))
              (get-popn (:latest-yr-popn popn-with-mig) :popn 2015 0 "F"))))))
+
+(deftest looping-test-test
+  (testing "The output of the loop matches the R code."
+    (let [proj-clj-2015 (looping-test data-inputs {:first-proj-year 2014
+                                                   :last-proj-year 2015})
+          proj-r-2015 (:end-population output-2015)]
+      (is (fp-equals? (get-popn proj-r-2015 :popn 0 "F")
+                      (get-popn proj-clj-2015 :popn 2015 0 "F") 0.0001)
+          (fp-equals? (get-popn proj-r-2015 :popn 90 "M")
+                      (get-popn proj-clj-2015 :popn 2015 90 "M") 0.0001)))))
```

Comment:
Suggest change this to a stronger test, which compares every value in the r and clj results